logic/cacheLogic.js

```diff
@@ -42,8 +42,8 @@ module.exports = class CacheLogic {
       await CacheLogic.getTipsAndVerifyLocalInfo();
       await CacheLogic.fetchChainNames();
       await CacheLogic.fetchPrice();
-      await aeternity.getOracleState();
       await CacheLogic.findContractEvents();
+      await aeternity.getOracleState();
```

Comment:
why change the order here?

Reply:
This was a test before I found the fix below, I will revert this.